src/content/docs/en/reference/publish-to-npm.mdx

```diff
@@ -233,10 +234,15 @@ Astro does not currently ship a test runner. _(If you are interested in helping
 
 In the meantime, our current recommendation for testing is:
 
+<Steps>
 1. Add a test `fixtures` directory to your `demo/src/pages` directory.
+
 2. Add a new page for every test that you'd like to run.
+
 3. Each page should include some different component usage that you'd like to test.
+
 4. Run `astro build` to build your fixtures, then compare the output of the `dist/__fixtures__/` directory to what you expected.
+</Steps>
 
 <FileTree>
```

Comment:
Feels like the file tree should be included in the steps 4 as it's the output that you should compare to, and this comparison is part of the step 4. Any thoughts?

Reply:
Yes! It wasn't in the original but I would say this does belong with Step 4, so let's include the file tree inside the steps.

Reply:
Moved inside steps

Reply:
Agreed, makes total sense